### PR TITLE
Adds requirements and gets tests passing

### PR DESCRIPTION
**Pull request by jamalhansen**
New requirements.txt and passing 11 of 11 tests.

Used pytest monkeypatching to avoid doing a requests.get()

### CODE REVIEW

File: unit_test.py [hunk 1]
```diff
@@ -19,70 +31,60 @@ def test_link_detect_finds_multiple_links():
...
-    assert sample_link['title'] == 'Test'
-    assert sample_link['markdown'] == '[Test](http://link.com)'
-
-def test_shownotes():
-    description = 'This is a test'
-    links = ['foo', 'bar']
-    notes = shownotes(description=description, links=links)
-    assert notes == description + '\n\nfoo\nbar\n'
-
-def test_links_to_string():
```
Comment (kjaymiller) [inline unit_test.py]: I actually put this function back into the code. If you want you can do a pull in a couple minutes and the function will be in again `show noter.links_to_string()`

**Comment by jamalhansen** [comment]
ok I rebased and returned the test for links_to_string()

**Comment by jamalhansen** [comment]
@kjaymiller I rebased in your latest commit and am once again ready to merge

**Comment by jamalhansen** [comment]
Merged!

**Pull request merged**